desktop/release-notes.md

```diff
@@ -23,13 +23,98 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.11.0
+2022-07-28
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Added two new extensions ([vcluster](https://hub.docker.com/extensions/loftsh/vcluster-dd-extension) and [PGAdmin4](https://hub.docker.com/extensions/mochoa/pgadmin4-docker-extension)) to the Extensions Marketplace.
+- Ability to sort extensions has been added to the Extensions Marketplace.
+- No more feedback nagging. Thanks to all your feedback, we identified a bug that was causing some users to be asked for feedback too frequently. You should now only be asked for feedback twice a year.
+- Custom theme settings for Docker Desktop allow you to specify dark or light mode for Docker Desktop independent of your device settings. Fixes [docker/for-win#12747](https://github.com/docker/for-win/issues/12747)
+- Add new flag for Windows installer, `--no-windows-containers` to disable Windows containers integration.
+
```

Comment:
We should add the support for running on VMware ESXi and Azure in here. I know it could have worked on previous releases, but we're tying support to this release.

- Docker Desktop is now fully supported for Docker Business customers inside VMware ESXi and Azure VMs. See (reference to documentation) for details.

In fact, let's put this as the first item in the New section.

Reply:
Maybe in highlights as well then ?

Reply:
Sounds great, do you know where the documentation link needs to point?

Reply:
The new link will be https://docs.docker.com/desktop/vm-vdi/

Reply:
@stephen-turner @aevesdocker I've implemented this change now, let me know if it matches what you want.

Reply:
Just noticed that this link seems to have broken the build as it doesn't exist. Is there a different process for internal links like this?

Reply:
Yeh not to worry - the link is broken because the PR that has created that link has not been published yet. On release day, I'll publish that PR first and then this one.